plugin/pulp/plugin/models/importer.py

```diff
@@ -39,29 +37,22 @@ def sync(self):
         platform :class: `pulp.app.models.Importer` base attributes and any custom attributes
         defined by the subclass.
 
-        The model attributes were loaded from the database and then had the user specified override
-        config applied on top. Additionally the importer is read-only and prevents the saving of
-        changes to the Importer instance.
+        The importer instance should be treated as read-only and plugin writers should be cautious 
+        not to save any configuration changes to the database.
 
         Instantiation and calling of the sync method by the platform is roughly done with the
         following:
 
             1. The plugin provides an implementation called WidgetImporter which subclasses Importer
 
-            2. The user makes a call to sync widget_importer (say id=10) with some override config
+            2. The user makes a call to sync widget_importer (say id=10)
 
             3. The platform loads the saved
                 >>> wi = WidgetImporter.objects.get(id=10)
 
-            4. The platform puts the WidgetImporter into read-only mode
+            4. The platform attaches the worker's working_dir to the instance.
```

Comment:
s/attaches the worker's working_dir to the instance/sets the working_dir attribute on the instance/